jenkins/README.md

```diff
@@ -45,6 +45,13 @@ If the author is not in the whitelist but should be trusted then by adding a
 comment **add to whitelist** on the PR, the author will then be able to run the
 jobs on its own.
 
+### How to clean up leftover VMs
```

Comment:
This title is not exactly reflecting the content. Maybe "Cloud Resources cleanup" ?

Reply:
done